state/machine_internal_test.go

```diff
@@ -26,10 +26,10 @@ func (s *MachineInternalSuite) SetUpTest(c *gc.C) {
 var _ = gc.Suite(&MachineInternalSuite{})
 
 func (s *MachineInternalSuite) TestCreateUpgradeLockTxnAssertsMachineAlive(c *gc.C) {
-	arbitraryId := "1"
+	arbitraryID := "1"
```

Comment:
This is how I would normally name it, and how it should be according to idiomatic Go, but I have always kept with the convention use all over Juju of I(little)d.